assets/js/src/core/modules/asset/editor/image/tab-manager/tabs/properties/components/table/cells/select-cell/select-cell.tsx

```diff
@@ -0,0 +1,77 @@
+/**
+* Pimcore
+*
+* This source file is available under two different licenses:
+* - Pimcore Open Core License (POCL)
+* - Pimcore Commercial License (PCL)
+* Full copyright and license information is available in
+* LICENSE.md which is distributed with this source code.
+*
+*  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+*  @license    https://github.com/pimcore/studio-ui-bundle/blob/1.x/LICENSE.md POCL and PCL
+*/
+
+import { type DefaultCellProps } from '@Pimcore/components/grid/columns/default-cell'
+import React, { useEffect, useRef, useState } from 'react'
+import { useEditMode } from '@Pimcore/components/grid/edit-mode/use-edit-mode'
+import { type RefSelectProps, Select } from 'antd'
+
+export const SelectCell = (props: DefaultCellProps): React.JSX.Element => {
+  const { isInEditMode, disableEditMode, fireOnUpdateCellDataEvent } = useEditMode(props)
+  const [open, setOpen] = useState<boolean>(false)
+  const selectRef = useRef<RefSelectProps>(null)
+
+  useEffect(() => {
+    if (isInEditMode) {
+      setOpen(true)
+      selectRef.current?.focus()
+    }
+  }, [isInEditMode])
+
+  function saveValue (value: string): void {
+    fireOnUpdateCellDataEvent(value)
+    disableEditMode()
+  }
+
+  function getCellContent (): React.JSX.Element {
+    if (!isInEditMode) {
+      return (
+        <>
+          { props.getValue() }
+        </>
+      )
+    }
+
+    function onBlur (e: React.FocusEvent<HTMLInputElement>): void {
+      saveValue(e.target.value)
+    }
+
+    function onKeyDown (e: React.KeyboardEvent<HTMLInputElement>): void {
```

Comment:
Would be nice when we also could add the enter key here to save the value.
Like we do it for the text field.